src/main/resources/db/migration/all/20240502000000__create_cas_2_summary_view.sql

```diff
@@ -0,0 +1,22 @@
+-- create application summary view
+DROP VIEW IF EXISTS cas_2_application_summary;
```

Comment:
Using CREATE OR REPLACE requires that the new view includes the existing columns in the view to be replaced (but can add new ones).  Therefore a DROP is required to refresh the view.

Reply:
Regarding your question on adding new fields @Harriethw.  Running CREATE OR REPLACE will add additional columns, but any existing ones need to be there.  I've removed the DROP since I don't think it's necessary.